src/main/java/com/agency04/sbss/pizza/model/Customer.java

```diff
@@ -1,73 +1,68 @@
 package com.agency04.sbss.pizza.model;
 
+import javax.persistence.*;
+import java.util.List;
 import java.util.Objects;
 
+@Entity
 public class Customer {
 
-    private String firstName;
-    private String lastName;
+    @Id
     private String username;
-    private String phoneNumber;
-    private String address;
 
-    public Customer(String firstName, String lastName,String username, String phoneNumber, String address) {
-        this.firstName = firstName;
-        this.lastName = lastName;
-        this.username = username;
-        this.phoneNumber = phoneNumber;
-        this.address = address;
-    }
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name="customer_details_id")
+    private CustomerDetails customerDetails;
 
-    public String getFirstName() {
-        return firstName;
-    }
+    @OneToMany(mappedBy = "customer",
+            cascade = CascadeType.ALL)
+    private List<Delivery> deliveries;
```

Comment:
GET /api/customer/${username} vraca `Customer` objekt koji ima listu `Delivery`-a, a `Delivery` sadrzi `Customer` tako da ce reponse biti beskonacan zbog ciklicke ovisnosti. Da bi to maknula mozes recimo dodati  `@JsonIgnore` anotaciju iznad customera u `Delivery` klasi.